New metric: Share of Voice (#77)
chaoss/wg-value: metrics/share-of-voice.md

# Share of Voice

> Can we communicate the value of our community as a comparison to something of value by our supporting business?

Reviewer: This is one alternative that makes more sense to me. We can discuss.

For sure, we want to put `Question:` at the beginning.
```suggestion
Question: If a supporting business values the "share of voice" metric, how can the value of a community be expressed in those terms?
```

## Description

> Ratio of conversation occurring by or about our community in comparison to some baseline. Also described as: Visibility, chatter, social media presence, top-of-funnel impact.

Reviewer: ```suggestion
Ratio of conversation occurring by or about a community in comparison to a baseline. Also described as: Visibility, chatter, social media presence, or top-of-funnel impact.
```

## Objectives

The metric supports the perception of continued value from community work in order to continue to secure funding. It highlights the value driven by community work by comparing it to something already valued by the organization. 

## Implementation

> ### Prerequisites

Reviewer: I don't think we need this "subheading".

1. We would make it bold if we wanted to keep it because it is not one of the headings in our [metrics-template](https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md)
2. The next sentence clearly states that it describes prerequisites.
```suggestion
```

> Share of voice only works if you can agree to its implied assumptions ahead of time:

Reviewer: I'm going through to make the language simpler and sharper. Disregard if I change the meaning in any of these edits.

```suggestion
Share of voice works if everyone can agree to its implied assumptions:
```

> - The more conversation about you, the better (which requires accepting the adage ‘there is no such thing as bad publicity’).

Reviewer: ```suggestion
- More conversation about a community is good (adage: ‘there is no such thing as bad publicity’).
```

> - You have a public location worth measuring (public content like blogs or reviews, Twitter conversation, conference talks, StackOverflow tags).

Reviewer: ```suggestion
- The community has a public location worth measuring (e.g., blogs or reviews, Twitter conversation, conference talks, or StackOverflow tags).
```

> - There is a competitor that comes to mind, whether its internal competition where you compare community SOV to PR or AR efforts or external, where you focus on the top 3 companies in a given space (ex. comparing how N projects from X, Y, or Z companies compete for conversation around Data Science).

Reviewer: SOV = Share of Voice?
PR = Public Relation?
AR = ???

```suggestion
- The community has a competitor, either in the public or internal to an organization. For the latter case, Share of Voice can be compared to public relations or ?AR? efforts of internal competitors. It is suggested to focus on the top 3 competitors in a given space (e.g., comparing how open source communities from X, Y, or Z companies compete for conversation around Data Science).
```

project --> community, to stay consistent 
Rephrased the text a bit.

> - Your competitors are measurable in the same way you are, meaning you have a representative list of their activities similar to your own.

Reviewer: ```suggestion
- The community and competitors are measurable in the same way (i.e., a similar list of representative activities).
```

> - You have a trusted and repeatable system of measurement (ideally software used by multiple teams for multiple programs).

Reviewer: ```suggestion
- A trusted and repeatable system of measurement is used, ideally software used by multiple teams for multiple programs.
```

> If these statements are not applicable to your situation, abandon SOV immediately. It’s not for you at this moment. Rely on anecdotes or something else.

Reviewer: ```suggestion
If these prerequisites are not met, there may be more effective strategies than using Share of Voice to demonstrate value.
```

### Visualizations

One example made with PowerPoint:

> ![sov-metric](https://user-images.githubusercontent.com/1744971/71270812-1b27c580-2318-11ea-9188-0a79ec700348.png)

Reviewer: Please include the image as part of the PR so that it gets added to the repository.

### Tools Providing the Metric

> _This depends completely on what you've decide to measure._ Here are a few examples, which are far from an exhuastive list. Please add others as you find them.

Reviewer: ```suggestion
Examples:
```

Everything else is implied.

- [Google Search Console](https://search.google.com/) for relative search traffic.

> - [StackOverflow Data Explorer](https://data.stackexchange.com/stackoverflow/query/new) for SOV on StackExchange sites.

Reviewer: ```suggestion
- [StackOverflow Data Explorer](https://data.stackexchange.com/stackoverflow/query/new) for Share of Voice on StackExchange sites.
```

> - [Export Tweet](https://www.exporttweet.com/) to measure hashtag SOV.

Reviewer: ```suggestion
- [Export Tweet](https://www.exporttweet.com/) to measure hashtag Share of Voice.
```

> - [Hootsuite Analytics](https://hootsuite.com/platform/analytics) for broader social media SOV.

Reviewer: ```suggestion
- [Hootsuite Analytics](https://hootsuite.com/platform/analytics) for broader social media Share of Voice.
```

> - [Sysomos](https://sysomos.com/platform/analyze/) for social and press article SOV.

Reviewer: ```suggestion
- [Sysomos](https://sysomos.com/platform/analyze/) for social and press article Share of Voice.
```

### Data Collection Strategies

> The framing of what your measuring will always impact data collection. Here are a few examples of how you might frame the SOV analysis and then move to measure it. 

Reviewer: ```suggestion
The framing of how the Share of Voice will be used as a measurement impacts data collection decisions. Examples:
```

1. **Impact around an event**. To secure funding for participation at an industry conference, you present the impact our community activities will have on the visibility of your community in a given space. Given the influence of Twitter on technology perception, you offer to compare the SOV of you and your community around the event in comparison to your top 3 competitors. To measure, you use Export Tweet to download all data around one or more keywords. You then use a regular expression to count the % of conversation your community had relative to other communities and show that you had an impact. For best results, establish a baseline by downloading historical data around this topic, which is likely to show a growth in SOV due to your work.

> 2. **Value per budget**. Starting from a similar event strategy as #1, add an additional comparison. Collaborate with colleagues where you know how much they spend for "visibility" at conferences to show an additional angle to this measurement. The result will be something like: "we grew our percentage of SOV by 34% relative to our competitors, and we did so for X amount of money, which is 1/5th the cost of just hosting a booth or 1/3rd the cost of running an evening event." 

Reviewer: These examples speak to "you". Not sure if we would improve the metric by changing that.

## References

- [The Value of Community Metrics](https://mbbroberg.fun/the-value-of-community-metrics/) by [@mbbroberg](https://twitter.com/mbbroberg)
- [Community Metrics: Share of Voice](https://mbbroberg.fun/community-metrics-share-of-voice/) by [@mbbroberg](https://twitter.com/mbbroberg)
- [How to Measure Share of Voice: PPC, SEO & Social Media](https://www.brandwatch.com/blog/how-to-measure-share-of-voice/) from Brand Watch
- [How to Calculate Share of Voice for Organic Search](https://moz.com/blog/how-to-calculate-share-of-voice-for-organic-search) from Moz 

> - [All about SOV](https://trackmaven.com/marketing-dictionary/share-of-voice/)

Reviewer: ```suggestion
- [All about Share of Voice](https://trackmaven.com/marketing-dictionary/share-of-voice/)
```
